src/structure_managers/structure_manager_centers.hh

```diff
@@ -271,6 +284,12 @@ namespace rascal {
       return 0;
     }
 
+    inline int get_atom_tag(const int & raw_index) const { return raw_index; }
+
+    inline size_t get_atom_tag(const size_t & raw_index) const {
+      return raw_index;
```

Comment:
Why raw index and why two functions one for size_t and one for int?

Reply:
atom_tag_list has int and somewhere I don't remember it is used to get_atom_tag so it is there to avoid compilation error

Reply:
See, this is also inconsistent with the earlier `decltype(auto) get_atom_tag()` change. We urgently need to agree on an interface for these things, otherwise these definitions are going to keep changing and introducing subtle bugs.

Reply:
This is more something that should be fixed at some point since fundamentally the function should not be needed. The fix is not trivial so it will not be fixed here.

Reply:
> Won't removing atom tags from the interface make a lot of things more complicated?

they should not be removed from since they are useful also in the ClusterRefs so don't worry.